uploadTool.py

```diff
@@ -0,0 +1,47 @@
+from bearuploader.Constants import classifiers, imports, encoding, requirements
+from bearuploader.Constants import setup_constants
+from tests.MockBear import MockBear
+from coalib.bears.requirements.PythonRequirement import PythonRequirement
+
+
+filename = "bearuploader/setup.py"
+
+bearOption = 'MockBear'
+bearName = bearOption + ".py"
+
+setup = open(filename, "w")
+reqs = open('bearuploader/requirements.txt', "w")
+
+Reqrs = MockBear.REQUIREMENTS
+
+for req in Reqrs:
+    if type(req) == PythonRequirement:
+        if req.version:
+            reqs.write(req.package + '==' + req.version + '\n')
+        else:
+            reqs.write(req.package + '\n')
+
+# set env
+setup.write('#!/usr/bin/env python3\n\n')
```

Comment:
does it harm if its here :D ? i think it helps when it is run as a script, to force python3 instead of python, as you do in bash scripting, not sure